e2e/app/portalregistry.go

```diff
@@ -252,7 +252,7 @@ func allowStagingValidators(ctx context.Context, def Definition) error {
 
 	contract, err := bindings.NewStaking(common.HexToAddress(predeploys.Staking), backend)
 	if err != nil {
-		return errors.Wrap(err, "new staking")
+		return errors.Wrap(err, "new portal registry")
```

Comment:
Without a broader context and looking only at the surrounding code, I do not understand why this error message text make sense. The staking contract is not the portal registry, is it?